v8py/jsobject.cpp

```diff
@@ -13,6 +13,7 @@ PyTypeObject js_object_type = {
 };
 PyMethodDef js_object_methods[] = {
     {"__dir__", (PyCFunction) js_object_dir, METH_NOARGS, NULL},
+    {"to_bytes", (PyCFunction) js_object_to_bytes, METH_NOARGS, NULL},
```

Comment:
This is problematic, it leaves no way to call a javascript method on the object called `to_bytes`.